client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -43,6 +46,67 @@ const CancelPurchaseRefundInformation = ( {
 		} );
 	};
 
+	const featuresList = ( productSlug, hasDomain ) => {
+		if ( typeof productSlug !== 'string' ) {
+			return null;
+		}
+
+		const planCancellationFeatures = getPlanCancellationFeatures( productSlug, hasDomain );
+
+		return (
+			<Fragment>
+				<div className="cancel-purchase__refund-information-description-header">
+					{ i18n.translate( 'When your %(productName)s plan expires you will loose access to:', {
```

Comment:
Should be "lose", not "loose".  Also probably a comma after "expires".

Also, it talks about "when your plan expires" but isn't this sentence also being displayed in the cancel-with-refund flow, when the plan will be removed immediately after canceling?  (I didn't test, but that's my sense from the screenshots.)

Reply:
I fixed the copy, do you want me to remove this string instead?

Reply:
Thanks!  I wouldn't necessarily remove it personally, just think that the wording doesn't really work for the case where the plan is being removed immediately (it does work fine for the non-refundable case where the cancellation just turns auto-renew off).  I'm not sure, but maybe something like "When your plan is removed" or "When your plan is cancelled" would work for the first case?